package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "open-msupply",
   "//": "Main version for the app, should be in semantic version format (any release candidate or test build should be separated by '-' i.e. 1.1.1-rc1 or 1.1.1-test",
-  "version": "1.5.00",
+  "version": "1.5.01",
```

Comment:
can you remove this before merging plz 🙏  
should be targeted for 1.6.0 now

Reply:
This goes to main, should it just be 1.5.0 ?

Reply:
oh, right. 1.5.0 then